src/__tests__/eventIntegration.test.ts

```diff
@@ -104,9 +104,17 @@ it('resolves eventsByIds correctly', async () => {
   const publisherId = 'publisherId';
 
   const getMock = jest.fn().mockResolvedValue({
-    id: eventId,
-    publisher: publisherId,
-  } as EventDetails);
+    data: [
```

Comment:
no new eventList test? or does this change cover it?

Reply:
Ok, we dont do any change to events-helsinki-ui, we just change this old one to use new api-endpoint.

Reply:
yeah